src/normalizer/chinese.rs

```diff
@@ -4,6 +4,9 @@ use super::CharNormalizer;
 use crate::detection::{Language, Script};
 use crate::normalizer::CharOrStr;
 use crate::Token;
+use kvariants::KVARIANTS;
+
+mod kvariants;
 
 /// Normalize Chinese characters by converting them into Pinyin characters.
```

Comment:
We should change this documentation

Reply:
Added b25c6d37d1e9b4dcf435c243dc1155cf28fcaf0a